src/containers/month.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import TaskGroup from './task-group';
+
+const Month = props => (
+  <li key={props.abbr} className='months-list__item'>
+    <div className='months-list-item__label-container'>
+      <span className='months-list-item__label' onClick={() => this.props.onMonthSelect}>
```

Comment:
Having read this again, I had a dyslexic moment and am failing to spot the additional () you added at the end of your query. I'll have to double check but I'm pretty sure whatever I've written will have syntactical issues.